### DISCUSSION TITLE
tiptap v2 codeView (HTML) extension?

### DISCUSSION THREAD

**Discussion by ag-coder** [ueberdosis/tiptap#1535] [Feature Requests]
I saw element-tiptap has a codeview extension. 
it would be great if tiptap v2  also has such an extension.
is it possible to rewrite the extension of element-tiptap so that it works with tiptap version 2?

**Comment by hanspagel** [comment]
Thanks! That feels out of scope for the package, it’ll live outside the editor anyway. But you can take that example and just put the HTML content in a tab (or whatever you like):

https://www.tiptap.dev/guide/output#option-2-html

If you’d like to have the HTML editable (and using Vue), you can integrate that v-model example:

https://www.tiptap.dev/installation/vue2#5-use-v-model-optional

✌️

**Reply by ag-coder** [reply]
Hello thanks for your quick answer!
Ok i will experiment with it.

**Reply by bryanjtc** [reply]
Has anyone found a react version of the codeview extension from element-tiptap?
